pilot/pkg/model/envoyfilter.go

```diff
@@ -33,6 +33,7 @@ type EnvoyFilterWrapper struct {
 	Namespace        string
 	workloadSelector labels.Instance
 	Patches          map[networking.EnvoyFilter_ApplyTo][]*EnvoyFilterConfigPatchWrapper
+	Priority         int32
```

Comment:
I think this will help in resolving this  https://github.com/istio/istio/issues/41536 too with few more code changes

Reply:
So to be clear it does not fix 41536

Reply:
it does not fix #41536. Just enables it with few more code changes